tests/config/pytest_modules.yml

```diff
@@ -2307,6 +2307,13 @@ subread/featurecounts:
   - modules/subread/featurecounts/**
   - tests/modules/subread/featurecounts/**
 
+subworkflows/bam_stats_samtools: &subworkflows_bam_stats_samtools
+  - subworkflows/nf-core/bam_stats_samtools/**
+  - tests/subworkflows/nf-core/bam_stats_samtools/**
+  # - *samtools_stats
```

Comment:
Had to comment this out because Prettier and nf-core linting was failing. Not sure whether they need to be added back in.

Reply:
There are two options to get the tests to trigger:
1. Add the module anchors to the subworkflows (less things to update and maintain because we only have to update the module triggers)
2. Add the tags to the pytest workflow spec.

I think we stayed away from 2 in the past because it triggers all the subworkflows locally, but personally those are all things that should be tested, and I think with `nf-core modules test` we can just use the specific test name if we want to avoid that behavior locally for most users and then let the CI fail when they've at least got the module behaving. It's a bit more code to maintain and update, but it's very declarative, and less "clever" so I think it'll keep the barrier for understanding low and therefor keep the barrier for contributing low, which should be our goal here.

So I vote 2, less work on the nf-core tools side and then we avoid anchors.